node-module-system/a-wrapped-module.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-console */
+console.log('typeof exports:', typeof exports);
+console.log('value of exports:', exports.value);
```

Comment:
here you want to log the variables themselves and not their `.value` properties.

Reply:
![image](https://user-images.githubusercontent.com/57373172/92047071-00c43980-ed39-11ea-9102-a3ccd100ae7a.png)

Reply:
Okay, I was thinking that nothing was defined because they were not declared in the local scope. I need to read more.